src/App/VPSScan/Main.hs

```diff
@@ -27,6 +28,13 @@ vpsOpts = VPSOpts <$> runSherlockOpts <*> optional runIPROpts <*> syOpts <*> org
               projectIDOpt = strOption (long "project" <> metavar "String" <> help "Project ID")
               revisionIDOpt = strOption (long "revision" <> metavar "String" <> help "Revision ID")
 
+ninjaGraphOpts :: Parser NinjaGraphOpts
+ninjaGraphOpts = NinjaGraphOpts <$> ninjaDepsOpt <*> lunchTargetOpt <*> scotlandYardUrlOpt
+                 where
+                   ninjaDepsOpt = optional $ strOption (long "ninjadeps" <> metavar "STRING" <> help "Path to a generated ninja_deps file. This is only used to speed up testing and will not normally be used.")
```

Comment:
Where are these comments output? If they are public when running `-h` then it may be worth it to see if we can hide these from public use since the first, `ninjadeps`, is only used for testing.

Reply:
It looks you can just delete the `<> help "..."` part and nothing is output about that option. I'll make that change